app/controllers/organen/beheer/new.js

```diff
@@ -0,0 +1,24 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class MandatenbeheerBestuursorganenNewController extends Controller {
+  @service router;
+
+  @action
+  buildSourceTtl(instanceUri) {
+    const bestuurseenheid = this.model.bestuurseenheid;
+
+    return `<${instanceUri}> <http://data.vlaanderen.be/ns/besluit#bestuurt> <${bestuurseenheid.uri}> .`;
+  }
+
+  @action
+  cancel() {
+    this.router.transitionTo('mandatenbeheer.bestuursorganen');
+  }
+
+  @action
+  onCreate({ instanceId }) {
+    this.router.transitionTo('mandatenbeheer.bestuursorganen.edit', instanceId);
```

Comment:
These need to transition to the new routes, error: `The route mandatenbeheer.bestuursorganen.edit was not found`

Reply:
well spotted!